### PR TITLE
implement ansa photo archive management

### PR DESCRIPTION
**Pull request by petrjasek**
there is new delete action available for ansa
photo archive items.

SDANSA-402

### CODE REVIEW

File: client/extensions/ansa-archive/src/extension.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+const extension: IExtension = {
+    id: 'ansa-archive',
+    activate: (superdesk: ISuperdesk) => {
+        const result: IExtensionActivationResult = {
+            contributions: {
+                entities: {
+                    article: {
+                        getActionsBulk: (articles) => {
+                            const actions: IArticleActionBulk[] = [];
```
Comment (tomaskikutis) [inline client/extensions/ansa-archive/src/extension.ts:15]: It'd be nice to stick to `Array<IArticleActionBulk>` format. It's enforced by a linter on the client, but not here.

Comment (petrjasek) [reply client/extensions/ansa-archive/src/extension.ts:15]: what's the difference?

**Comment by petrjasek** [comment]
> Is spiking allowed for these external ansa items? It's a bit weird that external items appear in our internal lists, they probably shouldn't or should they? Does their structure match `IArticle`?

it's not our spiking, but using their api to delete those external files. I suppose they don't have UI to manage that archive, or it's in the old system they are trying to replace

File: client/extensions/ansa-archive/src/extension.ts (hunk 1 above)
Comment (tomaskikutis) [reply client/extensions/ansa-archive/src/extension.ts:15]: None, it's just a formatting choice.

**Comment by tomaskikutis** [comment]
It's a bit weird that external items appear in our internal lists, they probably shouldn't or should they? Does their structure match `IArticle`?

**Comment by petrjasek** [comment]
it does match. this is items we have available via custom search provider, it's like a proxy to external archive

File: client/extensions/ansa-archive/src/extension.ts (hunk 1 above)
Comment (petrjasek) [reply client/extensions/ansa-archive/src/extension.ts:15]: I see, ok makes sense to be consistent

**Pull request merged**